src/components/RestaurantPreview.tsx

```diff
@@ -0,0 +1,31 @@
+import "@buildo/bento-design-system/index.css";
+import "@buildo/bento-design-system/defaultTheme.css";
```

Comment:
You shouldn't need to import styling in every component, the import in `App` should be sufficient.